cg/meta/orders/ticket_handler.py

```diff
@@ -0,0 +1,138 @@
+import logging
+import re
+from typing import Optional
+
+from cg.apps.osticket import OsTicket
+from cg.exc import TicketCreationError
+from cg.server.schemas.order import OrderIn
+from cg.store import Store, models
+
+LOG = logging.getLogger(__name__)
+
+
+class TicketHandler:
+    """Handle tickets in the meta orders context"""
+
+    NEW_LINE = "<br />"
```

Comment:
NEW_LINE is declared in two places now. Move to constants=

Reply:
Ok, good point I'll fix that. I don't think it should be in constants.py though since it is super specific for creating a ticket message.